src/main/java/com/burda/hw18/Xml.java

```diff
@@ -0,0 +1,32 @@
+package com.burda.hw18;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.util.Objects;
+
+public class Xml {
```

Comment:
Can be combined with the JSON class, they have the same code, only the regular expressions are different